src/main/resources/feedbackQuestionRubricEditFormTableOptionsFragment.html

```diff
@@ -0,0 +1,16 @@
+<td class="align-center rubricCol-${questionIndex}-${col}" data-col="${col}">
+    <div class="btn-group">
+        <button type="button" class="btn btn-default" id="rubricMoveChoiceLink-${questionIndex}-${col}-l" onclick="swapRubricCol(${col}, ${questionIndex}, true)"
+            data-toggle="tooltip" data-placement="top" title="Move column left">
```

Comment:
Lines that are intended to be a continuation of a previous line should have 8 spaces, to avoid confusion between this and inner elements.